Makefile

```diff
@@ -240,41 +242,28 @@ clean_certs:
 ###                  Formatting, linting, and vetting                       ###
 ###############################################################################
 
-# https://github.com/cometbft/cometbft/pull/1925#issuecomment-1875127862
-# Revisit using format after CometBFT v1 release and/or after 2024-06-01.
-#format:
-#	find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs gofmt -w -s
-#	find . -name '*.go' -type f -not -path "*.git*"  -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/cometbft/cometbft
-#.PHONY: format
-
-#? lint: Run latest golangci-lint linter
+#? lint: Lint, format and fix typos
 lint:
-	@echo "--> Running linter"
-	@go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run
+	@echo "--> Linting"
+	@go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run --fix
+	@echo "--> Formatting"
+	@go run mvdan.cc/gofumpt@latest -l -w ./..
```

Comment:
Not sure if I understood why `./..`.

Reply:
Moreover, for some reason it reports here files inside `.git` dir.

Reply:
fixed https://github.com/cometbft/cometbft/pull/2049/commits/fe80aad63d0095c95257399ac22441d7dabd0fe6